src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/MLDsa/MLDsaTestsData.Ietf.cs

```diff
@@ -0,0 +1,1592 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.DotNet.XUnitExtensions;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    public static partial class MLDsaTestsData
+    {
+        internal static partial MLDsaKeyInfo IetfMLDsa44 => field ??= new MLDsaKeyInfo(
+            MLDsaAlgorithm.MLDsa44,
+                                 "d7b2b47254aae0db45e7930d4a98d2c97d8f1397d17" +
+            "89dafa17024b316e9bec94fc9946d42f19b79a7413bbaa33e7149cb42ed51156" +
```

Comment:
Why offset at the beginning like this?

Reply:
It's how it looks in the IETF doc (it's in the middle of an ASN pretty print). Flushing to the left looked weird too so I just kept it like this.